Scripts/Miscellaneous/Unix_to_Human/timestamp_to_human.py

```diff
@@ -0,0 +1,5 @@
+import time
+
+unix_timestamp = int(input())
+utc_time = time.gmtime(unix_timestamp)
```

Comment:
before passing the UNIX timestamp object validate timestamp

Reply:
Can you please elaborate this? Thank you.